images/virtualization-artifact/pkg/controller/vd/vd_controller.go

```diff
@@ -99,20 +99,20 @@ func NewController(
 		return nil, err
 	}
 
-	vdcolelctor.SetupCollector(&lister{cache: mgr.GetCache()}, metrics.Registry)
+	vdcolelctor.SetupCollector(&lister{uc: unsafe.NewClient(mgr.GetCache())}, metrics.Registry)
```

Comment:
typo: vdcolelctor -> vdcollector

```suggestion
	vdcollector.SetupCollector(&lister{uc: unsafe.NewClient(mgr.GetCache())}, metrics.Registry)
```